.eslintignore

```diff
@@ -78,852 +78,852 @@ packages/react-native-vosk/lib/
-packages/renderer/MdToHtml/renderMedia.js
-packages/renderer/MdToHtml/rules/checkbox.js
-packages/renderer/MdToHtml/rules/code_inline.js
-packages/renderer/MdToHtml/rules/fence.js
-packages/renderer/MdToHtml/rules/fountain.js
-packages/renderer/MdToHtml/rules/highlight_keywords.js
-packages/renderer/MdToHtml/rules/html_image.js
-packages/renderer/MdToHtml/rules/image.js
-packages/renderer/MdToHtml/rules/katex.js
-packages/renderer/MdToHtml/rules/link_close.js
-packages/renderer/MdToHtml/rules/link_open.js
-packages/renderer/MdToHtml/rules/mermaid.js
-packages/renderer/MdToHtml/rules/sanitize_html.js
-packages/renderer/MdToHtml/rules/source_map.js
-packages/renderer/MdToHtml/setupLinkify.js
-packages/renderer/MdToHtml/validateLinks.js
-packages/renderer/headerAnchor.js
-packages/renderer/highlight.js
-packages/renderer/htmlUtils.js
-packages/renderer/htmlUtils.test.js
-packages/renderer/index.js
-packages/renderer/noteStyle.js
-packages/renderer/pathUtils.js
-packages/renderer/utils.js
-packages/tools/build-release-stats.js
-packages/tools/build-welcome.js
-packages/tools/buildServerDocker.js
-packages/tools/buildServerDocker.test.js
-packages/tools/bundleDefaultPlugins.js
-packages/tools/bundleDefaultPlugins.test.js
-packages/tools/checkIgnoredFiles.js
-packages/tools/checkLibPaths.js
-packages/tools/checkLibPaths.test.js
-packages/tools/convertThemesToCss.js
-packages/tools/generate-database-types.js
-packages/tools/generate-images.js
-packages/tools/git-changelog.js
-packages/tools/git-changelog.test.js
-packages/tools/licenseChecker.js
-packages/tools/release-android.js
-packages/tools/release-cli.js
-packages/tools/release-electron.js
-packages/tools/release-ios.js
-packages/tools/release-plugin-repo-cli.js
-packages/tools/release-server.js
-packages/tools/setupNewRelease.js
-packages/tools/spellcheck.js
-packages/tools/tagServerLatest.js
-packages/tools/tool-utils.js
-packages/tools/update-readme-download.js
-packages/tools/update-readme-download.test.js
-packages/tools/update-readme-sponsors.js
-packages/tools/updateMarkdownDoc.js
-packages/tools/utils/discourse.js
-packages/tools/utils/loadSponsors.js
-packages/tools/utils/translation.js
-packages/tools/website/build.js
-packages/tools/website/buildTranslations.js
-packages/tools/website/updateDownloadPage.js
-packages/tools/website/updateNews.js
-packages/tools/website/utils/applyTranslations.js
-packages/tools/website/utils/applyTranslations.test.js
-packages/tools/website/utils/convertLinksToLocale.js
-packages/tools/website/utils/convertLinksToLocale.test.js
-packages/tools/website/utils/frontMatter.js
-packages/tools/website/utils/news.js
-packages/tools/website/utils/openGraph.js
-packages/tools/website/utils/openGraph.test.js
-packages/tools/website/utils/parser.js
-packages/tools/website/utils/pressCarousel.js
-packages/tools/website/utils/processTranslations.js
-packages/tools/website/utils/render.js
-packages/tools/website/utils/types.js
+packages\app-cli\app\LinkSelector.js
```

Comment:
Ok but that's not good, you see it's converted all the `/` to `\`. It's also not sorting the files in the same way. This diff didn't look wrong to you?

It seems the script doesn't work for Windows and needs to be fixed

Reply:
Yes, to be honest, it just seems wrong to me. But, I don't know, I just ran the command and committed. That's why I asked you how I should proceed with this. I figured if it have any problems you'd check it out at some point and leave a comment, before accepting my PR, of course. Well, sorry, I believe I should have been more specific in my question.